geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxy.java

```diff
@@ -1934,6 +1948,10 @@ public Version getVersion() {
     return this.clientVersion;
   }
 
+  protected Subject getSubject() {
```

Comment:
probably mark this as @VisibleForTesting

Reply:
Thanks for the suggestion. Fixed.